switchlink/switchlink_route_test.cc

```diff
@@ -1,5 +1,19 @@
-// Copyright 2023-2024 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
+/*
+ * Copyright 2023-2024 Intel Corporation.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Do we need this blob? Doesnt the `SPDX` in line 3 cover the licensing agreement?

Reply:
I don't know. I haven't looked over the SWLC guideline lately.